src/components/MapEditor/index.js

```diff
@@ -58,11 +56,23 @@ const MapEditor = ({ location }) => {
   );
 };
 
+
 MapEditor.propTypes = {
   location: PropTypes.shape({
-    geometrie: PropTypes.shape({}),
-    address: PropTypes.shape({}),
+    geometrie: PropTypes.shape({
+      type: PropTypes.string,
+      coordinates: [PropTypes.number, PropTypes.number],
+    }),
+    address: PropTypes.shape({
+      openbare_ruimte: PropTypes.string,
+      huisnummer: PropTypes.string,
+      huisletter: PropTypes.string,
+      huisnummer_toevoeging: PropTypes.string,
+      postcode: PropTypes.string,
+      woonplaats: PropTypes.string,
+    }),
   }).isRequired,
+  options: PropTypes.shape({ }).isRequired,
```

Comment:
no, is not needed, the options shape is passed to the map and not used explicit here. We are talking about all the the all possible options of leaflet btw

Reply:
Can you then put in a docblock that has a link to the proper Leaflet documentation? Because, in the current case, there is no way to see which options can be passed in.

Reply:
ok